### PR TITLE
[data-help][s]: updated help message with headings and added data -h …

### PR DESCRIPTION
**Pull request by Mikanebu**
* updated headings in help message
* added `data -h` command

### CODE REVIEW

File: docs/get.md [hunk 1]
```diff
@@ -10,14 +10,13 @@ URL can be one of:
 * dataset in GitHub (e.g., https://github.com/datasets/co2-ppm)
 * direct URL to dataset
 
-## Options:
+**Options:**
```
Comment (rufuspollock) [inline docs/get.md]: Why are we switching from headings to bold? Having markdown headings is nice if we ever autogenerate documentation. Please can you state why the change in the commit message.

Comment (Mikanebu) [reply docs/get.md]: @rufuspollock because our markdown processor could not process headings into terminal. It will show `## Options` when you run `data get --help`. Plus, I updated according to `data push` help message to have all of them in same structure

Comment (rufuspollock) [reply docs/get.md]: But ## Options is what it should show ...

OK, i'm happy to have this for now 😄

Comment (anuveyatsu) [reply docs/get.md]: @Mikanebu misunderstanding here is that you expected "## Some text" to be rendered as large text like it does on hackmd etc. But it is rendered differently in the terminal so "##" is still there to show it is heading level 2 (notice it has different text color). Basically, you didn't need to change this but it's OK.

You can see example here - https://www.npmjs.com/package/marked-terminal#example

**Review by anuveyatsu — APPROVED** [review]
Looks good

File: docs/get.md (hunk 1 above)
Comment (rufuspollock) [reply docs/get.md]: @anuveyatsu ok, in that case can we change back to having ## stuff ... ?

@anuveyatsu i'll let you call this. I'd like to get this consistently right going forward so worth getting right now.

**Review by anuveyatsu — APPROVED** [review]
Good
